src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/AuroraTopologyService.java

```diff
@@ -165,8 +163,7 @@ public void setClusterInstanceTemplate(HostInfo clusterInstanceTemplate) {
     this.log.logTrace(Messages.getString(
         "AuroraTopologyService.2",
         new Object[] {clusterInstanceTemplate.getHost(),
-            clusterInstanceTemplate.getPort(),
```

Comment:
The database was removed from the instance template because it's one of the properties that shouldn't be shared between connections to the same cluster. Good catch with the error message though